real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/planners/DeadreckoningPlanner.java

```diff
@@ -0,0 +1,59 @@
+package com.roboclub.robobuggy.nodes.planners;
+
+import com.roboclub.robobuggy.messages.BrakeControlMessage;
+import com.roboclub.robobuggy.messages.DriveControlMessage;
+import com.roboclub.robobuggy.messages.GPSPoseMessage;
+import com.roboclub.robobuggy.messages.SteeringMeasurement;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.ros.Publisher;
+
+import java.util.Date;
+
+/**
+ * Created by vivaanbahl on 1/27/16.
+ */
+public class DeadreckoningPlanner extends PathPlannerNode {
+
+    private double currentCommandedSteeringAngle = 0;
+
+    /**
+     * Construct a new {@link PathPlannerNode}
+     *
+     * @param channel {@link NodeChannel} on which to broadcast status
+     *                information about the node
+     */
+    public DeadreckoningPlanner(NodeChannel channel)  {
+        super(channel);
+ 
+    }
+
+    @Override
+    protected void updatePositionEstimate(GPSPoseMessage m) {
+        // do nothing here, this is just a simple sweep
```

Comment:
This is ok since the position is not needed for the sweep.  It is meant to be a dumb tester case